### PR TITLE
CB-8498: On windows import from library shall return native URI when requested.

### PR DESCRIPTION
**Pull request by vldmrrr**

### CODE REVIEW

**Comment by Bubkastor** [comment]
Image not open this uri object.

File: src/windows/CameraProxy.js [hunk 1]
```diff
@@ -165,7 +165,16 @@ module.exports = {
 
                             var storageFolder = Windows.Storage.ApplicationData.current.localFolder;
                             file.copyAsync(storageFolder, file.name, Windows.Storage.NameCollisionOption.replaceExisting).then(function (storageFile) {
-                                successCallback(URL.createObjectURL(storageFile));
+                                var res;
+                                if (destinationType == Camera.DestinationType.NATIVE_URI) {
+                                    res = storageFile.path
+                                        .replace(/\\/g, '/')
+                                        .replace(':','%3A');
```
Comment (Bubkastor) [inline src/windows/CameraProxy.js]: there is no need to replace

Comment (vldmrrr) [inline src/windows/CameraProxy.js]: Colon is invalid character in URI. This replacement makes returned URI compatible with native URI suitable for File plugin.

File: src/windows/CameraProxy.js [hunk 2]
```diff
@@ -165,7 +165,16 @@ module.exports = {
 
                             var storageFolder = Windows.Storage.ApplicationData.current.localFolder;
                             file.copyAsync(storageFolder, file.name, Windows.Storage.NameCollisionOption.replaceExisting).then(function (storageFile) {
-                                successCallback(URL.createObjectURL(storageFile));
```
Comment (Bubkastor) [inline src/windows/CameraProxy.js]: Maybe just 
return successCallback("ms-appdata:///local/" + storagefile.name);

Comment (vldmrrr) [inline src/windows/CameraProxy.js]: Yeah, that should work as well. Just need to test.

Comment (vldmrrr) [inline src/windows/CameraProxy.js]: OK, tested, works, amended commit as suggested

**Pull request closed without merging**